README.md

```diff
@@ -49,73 +49,50 @@ account.
-| 5.0.0-M2               | 7.0.0-M3   |         |         |          |          |          | 7.0.0     |
-| 5.0.0                  | 7.0.0      |         |         |          |          |          | 7.0.0     |
-| [5.0.1][5.0.1]         | 7.0.0      |         |         |          |          |          | 7.0.0     |
-| 5.1.0-M1               | 7.1.0-M1   |         |         |          |          |          | 7.1.0-M1  |
-| 5.1.0-M2               | 7.1.0-M2   |         |         |          |          | 7.0.1    | 7.1.0-M2  |
-| 5.1.0                  | 7.1.0.1    |         |         |          |          | 7.0.1    | 7.1.0     |
-| [5.1.1][5.1.1]         | 7.1.0.1    |         |         |          |          | 7.0.1    | 7.1.0     |
-| 5.2.0-M1               | 7.2.0-M1   |         |         |          | 7.1.0.1  | 7.0.1    | 7.2.0-M1  |
-| 5.2.0-M2               | 7.2.0-M2   |         |         |          | 7.1.1    | 7.0.1    | 7.2.0-M2  |
-| [5.2.0][5.2.0]         | 7.2.0      |         |         |          | 7.1.1    | 7.0.1    | 7.2.0     |
-| 5.3.0-M1               | 23.1.0-M1  |         |         | 7.2.1    | 7.1.1.5  | 7.0.1.3  | 23.1.0-M1 |
-| 5.3.0-M2               | 7.3.0-M1   |         |         | 7.2.1    | 7.1.1.5  | 7.0.1.3  | 7.3.0-M2  |
-| 5.3.0-M3               | 7.3.0-M2   |         |         | 7.2.1    | 7.1.1.5  | 7.0.1.3  | 7.3.0-M2  |
-| [5.3.0][5.3.0]         | 7.3.0      |         |         | 7.2.1    | 7.1.1.5  | 7.0.1.3  | 7.3.0     |
-| 5.4.0-M1               | 7.4.0-M1   |         | 7.3.0.1 | 7.2.1.5  | 7.1.1.7  | 7.0.1.9  | 7.4.0-M1  |
-| 5.4.0-M2               | 7.4.0-M2   |         | 7.3.1   | 7.2.1.7  | 7.1.1.8  | 7.0.1.10 | 7.4.0-M2  |
-| 5.4.0-M3               | 7.4.0-M3   |         | 7.3.1   | 7.2.1.7  | 7.1.1.8  | 7.0.1.10 | 7.4.0-M3  |
-| [6.0.0][6.0.0]         | 7.4.0.1    |         | 7.3.1   | 7.2.1.11 | 7.1.1.8  | 7.0.1.10 | 7.4.0.1   |
-| [6.0.1][6.0.1]         | 7.4.0.1    |         | 7.3.1   | 7.2.1.11 | 7.1.1.8  | 7.0.1.10 | 7.4.0.1   |
-| [6.0.2][6.0.2]         | 7.4.0.1    |         | 7.3.1   | 7.2.1.11 | 7.1.1.8  | 7.0.1.10 | 7.4.0.1   |
-| [6.1.0-M.1][6.1.0-M.1] | 23.1.0-A19 | 7.4.0.1 | 7.3.1   | 7.2.1.11 | 7.1.1.8  | 7.0.1.10 | 7.4.0.1   |
-| [7.0.0-M.1][7.0.0-M.1] | 23.1.0-A27 | 7.4.1   | 7.3.1   | 7.2.1.12 | 7.1.1.8  | 7.0.1.10 | 7.4.1     |
-| [7.0.0-M.2][7.0.0-M.2] | 23.1.0-M4  | 7.4.1.1 | 7.3.1   | 7.2.1.12 | 7.1.1.8  | 7.0.1.10 | 7.4.1.1   |
-| [7.0.0][7.0.0]         | 23.1.0     | 7.4.1.2 | 7.3.1.1 | 7.2.1.12 | 7.1.1.8  | 7.0.1.10 | 23.1.0    |
-| [7.0.1][7.0.1]         | 23.1.1     | 7.4.1.3 | 7.3.1.2 | 7.2.1.13 | 7.1.1.10 | 7.0.1.10 | 23.1.0    |
-| [7.0.2][7.0.2]         | 23.1.1     | 7.4.1.3 | 7.3.1.2 | 7.2.1.13 | 7.1.1.10 | 7.0.1.10 | 23.1.0    |
-| [7.0.3][7.0.3]         | 23.1.1     | 7.4.1.3 | 7.3.1.2 | 7.2.1.13 | 7.1.1.10 | 7.0.1.10 | 23.1.0    |
-| [8.0.0-M.1][8.0.0-M.1] | 23.2.0-M1  | 7.4.1.3 | 7.3.1.2 | 7.2.1.13 | 7.1.1.10 | 7.0.1.10 | 23.1.0    |
```

Comment:
maybe we should keep it for reference? maybe in a subpage

Reply:
I don't think it has a lot of value and is probably more keeping people away from using the latest chart should they want to deploy an earlier version of the platform.